README.md

```diff
@@ -15,6 +15,17 @@
 
 The Unified Memory Framework (UMF) is a library for constructing allocators and memory pools. It also contains broadly useful abstractions and utilities for memory management. UMF allows users to manage multiple memory pools characterized by different attributes, allowing certain allocation types to be isolated from others and allocated using different hardware resources as required.
 
+**⚠️ Work-In-Progress Disclaimer:** \
+Recipient understands, acknowledges and agrees that the Software:
+> - is a pre-production, research version intended for use in a testing simulator;
```

Comment:
"simulator -> environment" would be better?

Reply:
I re-phrased the first sentence.